Makefile

```diff
@@ -21,6 +21,7 @@ INCLUDES= \
 	-I$(3RD_PARTY)/wren-0.1.0/src/include \
 	-I$(3RD_PARTY)/moonscript \
 	-I$(3RD_PARTY)/fennel \
+	-I$(3RD_PARTY)/curl \
```

Comment:
The 3rd-party submodule doesn't contain `curl`, so this might not work... 🤔

Reply:
You should do `submodule update` I think.

Reply:
Memo:
I got this error for `submodule update`
```
error: Server does not allow request for unadvertised object 82bacbf88361c77455cb8b3052d3c07372be13b1
Fetched in submodule path '3rd-party', but it did not contain 82bacbf88361c77455cb8b3052d3c07372be13b1. Direct fetching of that commit failed.
```

It seems that the current `3rd-party` submodule is using the original one.
Perhaps we need to:
- fork the `3rd-party` submodule in `tic80hackathon`
- set the forked `3rd-party` to be this `TIS-80`'s submodule
- add curl dependency in the forked one